packages/runtime-dom/src/modules/events.ts

```diff
@@ -57,16 +57,17 @@ export function patchEvent(
   }
 }
 
-const optionsModifierRE = /(?:Once|Passive|Capture)$/
+const optionsModifierRE =
+  /^(?!onOnce|onPassive|onCapture).*?(Once|Passive|Capture)$/
```

Comment:
That will help with `once` event, but not for example with `anotherOnce`.

`@anotherOnce` converts to name `onAnotherOnce`
`@another.once` (with modifier) also converts to name `onAnotherOnce`

Reply:
The process of converting `@another.once` to `onAnotherOnce` is unrelated to this context and does not affect the execution of the `onAnotherOnce` event. Regardless, `onAnotherOnce` will ultimately be transformed into `another {once: true}` in any case.